devices/src/bus.rs

```diff
@@ -253,4 +264,12 @@ mod tests {
         assert!(bus_clone.read(0x10, &mut data));
         assert_eq!(data, [1, 2, 3, 4]);
     }
+
+    #[test]
+    fn test_debug_error() {
```

Comment:
This actually tests `display` not `debug`.

Reply:
done